### PR TITLE
card

### PR DESCRIPTION
**Pull request by Hoa0304**
image

### CODE REVIEW

File: src/helper/random.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+function random(){
+    return Date.now()
+}
```
Comment (PhamNhuQuocBao) [inline src/helper/random.js:3]: format code (endOfLine).

File: src/views/components/card.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                    ${Book.name}
+                </span>
+                <br>
+                <span class="card__text-secondary">
+                    ${Book.author},${Book.year}
+                </span>
+                <p class="card__text--evaluate">
+                    ${Book.storRate}
+                    <span class="card__text--evaluate-extra">
+                        /5
```
Comment (PhamNhuQuocBao) [inline src/views/components/card.js:16]: /5 ???

File: src/views/layouts/sidebar.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            <li>
+                <img src="${book}" alt="books">
+                <a class="sidebar__menu--item-text" href="#">
+                    <span>My Shelf</span>
+                </a>
+            </li>
+        </ul>
+    </div>
+    <div class="sidebar__final">
+    <ul>
```
Comment (PhamNhuQuocBao) [inline src/views/layouts/sidebar.js:32]: format code.

File: src/views/components/card.js (hunk 2 above)
Comment (Hoa0304) [reply src/views/components/card.js:16]: image
I set the default to /5 because the star rating section always remains fixed at /5

**Pull request merged**